addons-dev/aldryn-cms/aldryn_config.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from aldryn_client import forms
+
+
+class Form(forms.BaseForm):
+    name = forms.CheckboxField('Name', required=False)
+
+    def to_settings(self, data, settings):
+        # TODO: break out a lot of this stuff into other Addons
```

Comment:
Are you going to add apphook-reload here? Or is this functionality supported on another level by the platform?